apps/backoffice-v2/src/domains/business-reports/adapters/report-adapter/report-adapter.ts

```diff
@@ -17,10 +19,20 @@ const toRiskLabels = (riskIndicators: Array<{ name: string; riskLevel: string }>
 
   return riskIndicators.map(({ name, riskLevel }) => ({
     label: name,
-    severity: riskLevel,
+    severity:
```

Comment:
The unified API returns severity of "positive" instead of "low" and there is a mix up between "medium" and "moderate" so for now its mapped in the backoffice.